testdata/not-the-leader/state.json.golden

```diff
@@ -0,0 +1,85 @@
+{
+   "Healthy": false,
+   "FailureTolerance": 0,
+   "Servers": {
+      "7875975d-d54b-49c1-a400-9fefcc706c67": {
+         "Server": {
+            "ID": "7875975d-d54b-49c1-a400-9fefcc706c67",
+            "Name": "node1",
+            "Address": "198.18.0.1:8300",
+            "NodeStatus": "alive",
+            "Version": "1.9.0",
+            "Meta": null,
+            "RaftVersion": 3,
+            "IsLeader": true,
+            "NodeType": "voter",
+            "Ext": null
+         },
+         "State": "leader",
+         "Stats": {
+            "LastContact": 0,
+            "LastTerm": 3,
+            "LastIndex": 1024
+         },
+         "Health": {
+            "Healthy": true,
+            "StableSince": "2020-11-02T15:00:00Z"
+         }
+      },
+      "e72eb8da-604d-47cd-bd7f-69ec120ea2b7": {
+         "Server": {
+            "ID": "e72eb8da-604d-47cd-bd7f-69ec120ea2b7",
+            "Name": "node3",
+            "Address": "198.18.0.3:8300",
+            "NodeStatus": "alive",
+            "Version": "1.9.0",
+            "Meta": null,
+            "RaftVersion": 3,
+            "IsLeader": false,
+            "NodeType": "voter",
+            "Ext": null
+         },
+         "State": "voter",
+         "Stats": {
+            "LastContact": 15000000,
+            "LastTerm": 3,
+            "LastIndex": 500
+         },
+         "Health": {
+            "Healthy": false,
```

Comment:
This will only happen if we are using the correct term/index from the leader and not from the local server that is an unhealthy follower.